src/test/java/com/epam/tc/hw2/ex1/ExerciseFirstTest.java

```diff
@@ -0,0 +1,75 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BrowserTestBase;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+
+public class ExerciseFirstTest extends BrowserTestBase {
+    // 5. Assert that there are 4 items on the header section are displayed and they have proper texts
+    @Test
+    public void test4ItemsHeader() {
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(driver.findElement(By.partialLinkText("HOME")).getText())
+                .isEqualTo("HOME");
+        softAssertions.assertThat(driver.findElement(By.partialLinkText("CONTACT FORM")).getText())
+                .isEqualTo("CONTACT FORM");
+        softAssertions.assertThat(driver.findElement(By.partialLinkText("SERVICE")).getText())
+                .isEqualTo("SERVICE");
+        softAssertions.assertThat(driver.findElement(By.partialLinkText("METALS & COLORS")).getText())
+                .isEqualTo("METALS & COLORS");
+        softAssertions.assertAll();
+    }
+
+    // 6. Assert that there are 4 images on the Index Page and they are displayed
+    // 7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+    @Test
+    public void test4ImagesIndexPage() {
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(driver.findElements(By.className("benefit-txt")).get(0).getText())
+                .isEqualTo("To include good practices\n"
+                        + "and ideas from successful\n"
+                        + "EPAM project");
+        softAssertions.assertThat(driver.findElements(By.className("benefit-txt")).get(1).getText())
+                .isEqualTo("To be flexible and\n"
+                        + "customizable");
+        softAssertions.assertThat(driver.findElements(By.className("benefit-txt")).get(2).getText())
+                .isEqualTo("To be multiplatform");
+        softAssertions.assertThat(driver.findElements(By.className("benefit-txt")).get(3).getText())
+                .isEqualTo("Already have good base\n"
+                        + "(about 20 internal and\n"
+                        + "some external projects),\n"
+                        + "wish to get more…");
+        softAssertions.assertAll();
+    }
+
+    // 8. Assert that there is the iframe with “Frame Button” exist
+    // 9. Switch to the iframe and check that there is “Frame Button” in the iframe
+    // 10. Switch to original window back
+    @Test
+    public void testIframeWithButtonExists() {
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(driver.findElement(By.id("frame"))).isNotNull();
```

Comment:
isDisplayed()

Reply:
this method is not available

Reply:
Can not agree with you. It's available.

Reply:
![image](https://user-images.githubusercontent.com/98466902/213685500-96d12cc1-b685-49a7-a7e0-01186f78b0a1.png)

Reply:
Maybe, I need another changes, but I don`t notice now what I can change here more(